test/e2etest/kubetest/kubetest_runner.go

```diff
@@ -0,0 +1,231 @@
+package kubetest
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"github.com/gardener/test-infra/test/e2etest/config"
+	"github.com/pkg/errors"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+)
+
+type TestsKind string
+
+const (
+	PARALLEL TestsKind = "parallel"
+	SERIAL   TestsKind = "serial"
+)
+
+func init() {
+	kubectlPath, err := exec.Command("which", "kubectl").Output()
+	if err != nil {
+		log.Fatal(errors.Wrapf(err, "kubectl command unknown"))
+	}
+	if err = os.Setenv("KUBECTL_PATH", strings.TrimSpace(string(kubectlPath))); err != nil {
+		log.Fatal(errors.Wrapf(err, "Couldn't set environment variable KUBECTL_PATH"))
+	}
+	log.Debugf("KUBECTL_PATH: '%s'", os.Getenv("KUBECTL_PATH"))
+
+	if err = os.Setenv("KUBECONFIG", config.ShootKubeconfigPath); err != nil {
+		log.Fatal(errors.Wrapf(err, "Couldn't set environment variable KUBECONFIG"))
+	}
+	log.Debugf("KUBECONFIG: '%s'", os.Getenv("KUBECONFIG"))
+
+	if err = os.Setenv("KUBERNETES_CONFORMANCE_TEST", "y"); err != nil {
+		log.Fatal(errors.Wrapf(err, "Couldn't set environment variable KUBERNETES_CONFORMANCE_TEST"))
+	}
+
+	if err = os.Setenv("GINKGO_NO_COLOR", "y"); err != nil {
+		log.Fatal(errors.Wrapf(err, "Couldn't set environment variable GINKGO_NO_COLOR"))
+	}
+}
+
+// DryRun runs kubetest with dryrun=true argument
+func DryRun() (logDir string) {
+	kubetestArgs := createKubetestArgs("", false, true, 1)
+	runKubetest(kubetestArgs)
```

Comment:
shouldn't we get some status back from the run like an error if somthing failed?